src/status_im/ui/screens/stickers/views.cljs

```diff
@@ -33,17 +33,16 @@
                                                    (or no-snt? not-enough-snt?) nil
                                                    :else (re-frame/dispatch [:stickers/buy-pack id price]))}
        [react/view (styles/price-badge (and (not (or owned (zero? price))) (or no-snt? not-enough-snt?)))
-        (when (and (not (zero? price))) ;(not no-snt?))
-          [icons/icon :icons/price {:color colors/white :container-style {:margin-right 8}}])
+        (when (and (not (zero? price)) (not owned))
```

Comment:
rename it `owned?` please since it is a predicate

Reply:
I apologize since I'm new to Clojure, but where is it declared as a predicate? If I change it to `owned?` I get:

```sh
Figwheel: Compile Warning - Use of undeclared Var status-im.ui.screens.stickers.views/owned? in file src/status_im/ui/screens/stickers/views.cljs at line 36, column 45
```

Reply:
@igetgames my bad I didn't see you answered, you have to rename it everywhere in the scope, the scope here is the whole function because owned is one of the arguments, so it means you rename owned to owned? everywhere within `price-badge` function here.

there is no such thing as declaring it as a predicate, by predicate I mean that it is a value that we expect to be truthy or falsy (false or nil in clojure). and by convention they are named with a `?` at the end to emphasize that.

Reply:
Understood, completed in c37c9aa.